src/lake/Lake/CLI/Init.lean

```diff
@@ -156,6 +156,24 @@ def mathToolchainBlobUrl : String :=
 def mathToolchainUrl : String :=
   "https://github.com/leanprover-community/mathlib4/blob/master/lean-toolchain"
 
+def leanActionWorkflowContents :=
+s!"name: Lean Action CI
```

Comment:
Isn't this just a plain string? The `s!` doesn't seem necessary.

Reply:
Yes, `s!` was unnecessary. Removed.